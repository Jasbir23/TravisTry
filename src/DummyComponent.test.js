import "react-native";
import React from "react";
import DummyComponent from "./DummyComponent.js";
import renderer from "react-test-renderer";

test("renders correctly", () => {
  const tree = renderer.create(<DummyComponent />).toJSON();
  expect(tree).toMatchSnapshot();
});
