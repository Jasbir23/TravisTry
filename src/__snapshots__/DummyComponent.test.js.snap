exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Open up App.js to start working on your app!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Changes you make will automatically reload.
  </Text>
</View>
`;
